main/FloatingActionButtonWrapper/index.tsx

```diff
@@ -0,0 +1,120 @@
+import {DoobooTheme, withTheme} from '../theme';
+import {Icon, IconName} from '../Icon';
+import React, {FC, useMemo} from 'react';
+
+import {Animated} from 'react-native';
+import {IconButton} from '../IconButton';
+import styled from '@emotion/native';
+
+const AbsolutePositionWrapperView = styled.View`
+  position: absolute;
+`;
+
+export const StyledIcon = styled(Icon)`
+  color: ${({theme}) => theme.textContrast};
+`;
+
+const FABItemWrapperView = styled.View`
+  position: relative;
+  margin: 10px;
+`;
+
+export type FABItem = {
+  icon: IconName;
+  onPress: () => void;
+};
+
+interface FloatingActionButtonsWrapperProps {
+  active: boolean;
+  DefaultFAB: FABItem;
+  ActiveFAB: FABItem;
+  FABList: FABItem[];
+  renderDefaultFAB?: (item: FABItem) => React.ReactElement;
+  renderActiveFAB?: (item: FABItem) => React.ReactElement;
+  renderFABListItem?: (item: FABItem, idx: number) => React.ReactElement;
+  zIndex?: number;
+  top?: number;
+  bottom?: number;
+  left?: number;
+  right?: number;
```

Comment:
These prop should be wrapped with `style` object.
@hyochan wants components in `dooboo-ui` has `style` and `styles` props.
See other component(`Button` is good example) for more detail. 

Or maybe @hyochan can explain this better than me.

Reply:
thanks, I will change It to StyleProp<ViewStyle> and set the default value